lib/smack/SmackRep.cpp

```diff
@@ -1068,6 +1079,14 @@ std::string SmackRep::getPrelude() {
       << "\n";
   }
 
+  //s << "// Context-aware float-integer conversions" << "\n";
```

Comment:
Could we remove this completely instead of commenting it out?
I would prefer if we would do that.

Reply:
Can't you remove completely this commented out section?